package.json

```diff
@@ -20,7 +20,9 @@
   "scripts": {
     "build": "lerna exec yarn build --stream",
     "lint": "tslint 'packages/**/*.ts'",
+    "postinstall": "yarn build",
```

Comment:
added this, so that when people clone this repo and install, all the packages are built.